hub/core/compute/ray.py

```diff
@@ -19,6 +19,7 @@ def map(self, func, iterable):
     def close(self):
         self.pool.close()
         self.pool.join()
+        ray.shutdown()
```

Comment:
Bad design, will shutdown customer cluster upon execution completed. [This](https://github.com/activeloopai/Hub/pull/1412/files#diff-8d08f971bb09caf7fc13a3a5fcd95d00964658472cd9826ed1e839460d2acbc7R26-R35) should take care of ray in tests.

Reply:
yep agree removed